.github/ISSUE_TEMPLATE.md

```diff
@@ -2,7 +2,11 @@
 
 Output of `helm version`:
 
-Output of `kubectl version`:
+Output of `kubectl version`:  
+
+Output of `cat /etc/os-release`:  
+
+Output of `uname -r`:
```

Comment:
There are Linux specific commands. People use Helm on MacOS and Windows. Would you mind adding some language for those as well?